apps/web/playwright/organization/booking-list.e2e.ts

```diff
@@ -162,17 +162,19 @@ test.describe("Bookings list for organizations", () => {
 
       const eventType = await team1owner.getFirstTeamEvent(team1.id);
 
-      await expectPageToBeNotFound({ page, url: `/team/${team1.slug}/${eventType.slug}` });
+      // await expectPageToBeNotFound({ page, url: `/${commonUser.username}/${eventType.slug}` });
```

Comment:
for some reason the page is found for users in an org
maybe its because of the NEXT_PUBLIC_IS_E2E varible we set in .env

Reply:
@hariombalhara @Udit-takkar Any ideas on this one?

Reply:
@SomayChauhan why would the page be not found here?

Reply:
similar to https://github.com/SomayChauhan/cal.com/blob/634f42414f58a3ae20b04789dbaff318574cdd4d/apps/web/playwright/organization/booking.e2e.ts#L53